app/models/mhv_account.rb

```diff
@@ -116,6 +119,21 @@ def already_premium?
 
   private
 
+  def track_state
+    if INELIGIBLE_STATES.include?(aasm(:account_state).to_state)
+      tracker = MHVAccountIneligible.find(user.uuid)
+      return if tracker && tracker.account_state == aasm(:account_state).to_state
+      if tracker
+        tracker.update(account_state: aasm(:account_state).to_state)
+      else
+        attrs = { uuid: user.uuid, account_state: aasm(:account_state).to_state,
+                  mhv_correlation_id: mhv_correlation_id, icn: user.icn }
+        MHVAccountIneligible.create(attrs)
+      end
+      StatsD.increment("#{STATSD_ACCOUNT_INELIGIBLE_KEY}.#{aasm(:account_state).to_state}")
```

Comment:
@wyattwalter does my StatsD approach seem reasonable to you? This is one of the metrics Kam talked to you about recently.